client/src/App.js

```diff
@@ -18,7 +18,26 @@ function App() {
     of a JSX element will be read as text like HTML and not as JavaScript. */
   return (
     <div className="App">
-      <header className="App-header">
+      {/* Should we be including all this text in header? Should we also be using body? */}
+      <header className="App-header"> 
```

Comment:
This header tag is missing a closing tag, and causes an error when trying to run locally. I'll create a bug ticket to fix this since we already merged it in. But it's always a good idea to make sure your code compiles and runs before committing it.